src/com/epam/javatrainings/classdecomposition/service/IngredientService.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.javatrainings.classdecomposition.service;
+
+import com.epam.javatrainings.classdecomposition.persistance.Ingredient;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class IngredientService {
+    public static List<Ingredient> addPizzasIngredient(Ingredient ingredient){
+        List<Ingredient> ingredients = new ArrayList<>();
+        checkDuplicateIngredient(ingredient,ingredients);
+        return ingredients;
+    }
+
+    public static double calculatePizzasIngredientsCost(List<Ingredient> ingredients){
+        double pizzasIngredientsCost = 0;
+        for (Ingredient ingredient : ingredients) {
+            pizzasIngredientsCost += ingredient.getCost();
+        }
+        return pizzasIngredientsCost;
+    }
+
+    public static void printPizzasIngredients(List<Ingredient> ingredients){
+        for (Ingredient ingredient : ingredients) {
+            System.out.println(ingredient.getName() + " " + ingredient.getCost() + " €");
+        }
+    }
+
+    public static double checkIngredientFixedCost(String name) {
+        List<String> ingredientNames = new ArrayList<>(Arrays.asList("Tomato paste", "Cheese", "Salami", "Bacon", "Garlic", "Corn", "Pepperoni", "Olives"));
```

Comment:
again, ingredient can be a class or enum {name, price}

Reply:
I haven't used ENUM because before that task we haven't learned about ENUMs, I thought that I couldn't use in that case, then I asked from Levon if I can use some new features that we yet haven't learned and he replied  yes, but it's already late, because I have pushed my code before that